tests/integration/api_v3/txUtil/createTx.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+const { transactions } = require('@liskhq/lisk-client');
+const { cryptography } = require('@liskhq/lisk-client');
+const axios = require('axios');
```

Comment:
May be we can use `api` from helpers
```suggestion
const { api } = require('../../../helpers/api');
```